src/bundler.ts

```diff
@@ -267,12 +282,13 @@ class Bundler {
     // should be removed from the document.
     const importMap: Map<string, Import|null> = new Map();
     analyzedRoot.getByKind('import').forEach((i) => importMap.set(i.url, i));
-    this.excludes.forEach((u) => {
-      if (importMap.has(u)) {
+    importMap.forEach((_, u) => {
```

Comment:
I'm totally missing the O(n^2) -- This is an O(n) map population.  Maybe I am wearing blinders somehow?